nvflare/private/fed/app/server/runner_process.py

```diff
@@ -115,7 +118,7 @@ def start_server_training(server, args, app_root, run_number, snapshot):
 
         set_up_run_config(server, conf)
 
-        server.engine.set_run_number(run_number)
+        server.engine.creat_parent_connection(int(args.conn))
```

Comment:
typo here

Reply:
note that this requires the server.engine to be an instance of "ServerEngine"

Reply:
done